pkg/minikube/assets/addons.go

```diff
@@ -233,6 +233,13 @@ var Addons = map[string]*Addon{
 			"registry-creds-rc.yaml",
 			"0640"),
 	}, false, "registry-creds"),
+	"freshpod": NewAddon([]*BinDataAsset{
+		NewBinDataAsset(
+			"deploy/addons/freshpod/freshpod-dp.yaml",
```

Comment:
if you keep it as Deployment, `-deploy` might be a better suffix.

Reply:
No problem!!  I'll change to RC.